models/dex/dex_trades.sql

```diff
@@ -44,6 +44,7 @@ list of models using old generic test, due to multiple versions in one model:
 ,'wombat_bnb_trades'
 ,'iziswap_bnb_trades'
 ,'babyswap_bnb_trades'
+,'cow_protocol_trades'
```

Comment:
Is this supposed to be here? I thought we were only part of aggregator trades...

Reply:
ah, right, let me remove it